shared/src/model/response.model.ts

```diff
@@ -1,9 +1,10 @@
 import { Static, Type } from '@fastify/type-provider-typebox'
+import { Nullable } from './misc'
 
 export const PageInfoSchema = Type.Object({
   hasNextPage: Type.Boolean(),
   hasPreviousPage: Type.Boolean(),
-  startCursor: Type.String(),
-  endCursor: Type.String()
+  startCursor: Nullable(Type.String()),
+  endCursor: Nullable(Type.String())
```

Comment:
These will be null if the page of results returned is empty, which happens sometimes. You can simulate it by setting the 'before' query param to the result of the 'startCursor' from a query.